### DISCUSSION TITLE
multiple screens custom positins

### DISCUSSION THREAD

**Discussion by ghost** [rxhanson/RectanglePro-Community#628] [Ideas]
I am trying to make custom positions but it doesn't work for 2 displays (only if one of them is in portrait)
If I set a custom position/size and put it into the snap area it only applies to the display I took the position from. It makes sense but why can't we make 2 snap areas/for two separate displays?

**Comment by rxhanson** [comment]
If I understand correctly, I think you'll want to convert your custom size & position into relative values, which are between 0 and 1. That way, the size and position will be calculated based on the current display. Unfortunately, this would have to be done manually at this time. It's on my todo list to make an automated way to do this.

**Comment by ghost** [comment]
> If I understand correctly, I think you'll want to convert your custom size & position into relative values, which are between 0 and 1. That way, the size and position will be calculated based on the current display. Unfortunately, this would have to be done manually at this time. It's on my todo list to make an automated way to do this.

Sorry not a native speaker. I guess that's what I meant. Having to different snap area setups for each display would help too.

Off topic but isn't there a way to use any other keys but the modifier keys for moving and rezesing window? control option and command are fairly often used and it can be annoyed when I trigger a resize window or move window

**Comment by rxhanson** [comment]
> Off topic but isn't there a way to use any other keys but the modifier keys for moving and rezesing window? control option and command are fairly often used and it can be annoyed when I trigger a resize window or move window

It's on my backlog to allow a click + modifier key to initiate the move/resize. Is that what you're looking for?

**Comment by ghost** [comment]
> backlog

I guess it would already help if I can't set up a mouse button (like mouse button 4 or something) or if it would distinguish between left or right cmd etc.

**Comment by ghost** [comment]
so will you add mouse button options for this? It would also be great to be able to use move and resize with the trackpad

**Reply by rxhanson** [reply]
I don't have a timeframe for it, but it's on my backlog.
